src/mpid/ch4/src/ch4_impl.h

```diff
@@ -550,4 +550,12 @@ static inline uintptr_t MPIDI_CH4I_win_base_at_target(const MPIR_Win * win)
     return (uintptr_t) win->base;
 }
 
+static inline void MPIDI_find_tag_ep(MPIR_Comm* comm, int target_rank, int tag, int* ep_idx) {
+    *ep_idx = 0;
+}
+
+static inline void MPIDI_find_rma_ep(MPIR_Win* win, int target_rank, int* ep_idx) {
+    *ep_idx = 0;
+}
+
```

Comment:
I was going back and forth between 1) a single hash function that takes into account the type operation; and 2) a hash function per type of operation. I choose the latter to avoid extra branches.